Sharpmake.Generators/Rider/RiderJson.Util.cs

```diff
@@ -11,6 +11,7 @@ public static class CppLanguageStandard
         {
             public const string Cpp14 = "Cpp14";
             public const string Cpp17 = "Cpp17";
+            public const string Cpp20 = "Cpp20";
```

Comment:
Could you break out this change and submit a small PR with just this change? That one we can merge directly and doesn't have to be part of the Rider one.